GoogleSignIn/Sources/GIDSignInInternalOptions.h

```diff
@@ -23,9 +23,11 @@
 #endif
 
 #import "GoogleSignIn/Sources/GIDSignIn_Private.h"
+#import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h"
 
 @class GIDConfiguration;
 @class GIDSignInResult;
+@class GIDVerifiableAccountDetail;
```

Comment:
I don't think you need to forward declare here because you import the header. Related question: do you need to import the header?

Reply:
Removed the forward the declaration. As for the import, this is needed since I'm creating a completion block, GIDVerifyCompletion, which is created in the header and can't access through forward declaration.

Reply:
Got it; makes sense to me!